src/ansys/aedt/core/modeler/cad/object_3d.py

```diff
@@ -1096,15 +1096,11 @@ def name(self):
 
     @name.setter
     def name(self, obj_name):
-        if obj_name not in self._primitives.object_names:
-            if obj_name != self._m_name:
-                vName = []
-                vName.append("NAME:Name")
-                vName.append("Value:=")
-                vName.append(obj_name)
+        if obj_name != self._m_name:
+            if obj_name not in self._primitives.object_names:
```

Comment:
```suggestion
        if obj_name != self._m_name and obj_name not in self._primitives.object_names:
```
Why not joining the two if?

Reply:
Agreeing with Alberto comment